src/controllers/dlgprefcontroller.cpp

```diff
@@ -42,6 +42,13 @@ DlgPrefController::DlgPrefController(QWidget* parent,
     // Create text color for the file and wiki links
     createLinkColor();
 
+    // Check the text color of the palette for whether to use dark or light icons
+    if (!Color::isDimColor(palette().text().color())) {
```

Comment:
There are some unrelared changes in this branch... I guess I'll start from scratch.

Reply:
That was done for a reason. I don't remember exactly why, but I think you'll find out if you start over from scratch.

Reply:
ah ok, now I understand the rationale behind this.